Extension/src/LanguageServer/client.ts

```diff
@@ -379,6 +380,16 @@ class DefaultClient implements Client {
         }
 
         let abTestSettings: ABTestSettings = getABTestSettings();
+        
+        let intelliSenseCacheDisabled: boolean = false;
+        if (os.platform() === "darwin") {
+            const releaseParts: string[] = os.release().split(".");
+            if (releaseParts.length >= 1) {
+                // AutoPCH doesn't work for older Mac OS's.
+                intelliSenseCacheDisabled = parseInt(releaseParts[0]) < 18;
```

Comment:
I thought 17 (High Sierra) worked, no?

Reply:
Oops, you're right, I got my numbers confused.